doc: add ambassador message - benefits of Node.js (#56085)
nodejs/node: doc/contributing/advocacy-ambasador-program.md

Add a list of GitHub handles for those within the project that
have volunteered to be contacated when necessary by ambassadors
to get more info about the message to be promoted.

#### Node.js is a great choice for a JavaScript runtime

##### Goal

Highlight the benefits of chosing Node.js as your backend JavaScript runtime. Focus on what is great

> about Node.js avoiding comparisons with other optiona. We don't want to say negative things about

Reviewer: ```suggestion
about Node.js avoiding comparisons with other options. We don't want to say negative things about
```

other options, only highlight what is great about Node.js as a choice

Some of the things to highlight include:

> * How widely it is used (you never get fired for chosing Node.js)

Reviewer 2: All these sentences need a final dot.
```suggestion
* How widely it is used (you never get fired for chosing Node.js).
```

* The openess of the project. It is part of the OpenJS Foundation and its's governance is set up to avoid

>   any one company from dominating the project. Decisions are made by the collaboratora (of which there are quite

Reviewer: ```suggestion
  any one company from dominating the project. Decisions are made by the collaborators (of which there are quite
```

>   a few) s versus a small

Reviewer: ```suggestion
  a few)  versus a small
```

Add a list of GitHub handles for those within the project that
have volunteered to be contacated when necessary by ambassadors
to get more info about the message to be promoted.

#### Node.js is a great choice for a JavaScript runtime

##### Goal

Highlight the benefits of chosing Node.js as your backend JavaScript runtime. Focus on what is great

> about Node.js avoiding comparisons with other options. We don't want to say negative things about

Reviewer 3: nit: rephrase the comparison part?
```suggestion
about Node.js without drawing comparisons to alternatives. We don't want to say negative things about
```

> other options, only highlight what is great about Node.js as a choice

Reviewer 3: nit: missing comma
```suggestion
other options, only highlight what is great about Node.js as a choice.
```

Some of the things to highlight include:

* How widely it is used (you never get fired for chosing Node.js).

> * The openess of the project. It is part of the OpenJS Foundation and its's governance is set up to avoid

Reviewer 3: nit: typo
```suggestion
* The openess of the project. It is part of the OpenJS Foundation and it's governance is set up to avoid
```

  any one company from dominating the project. Decisions are made by the collaborators (of which there are quite

>   a few) versus a small
>   number of people.

Reviewer 3: nit: unnecessary newline?
```suggestion
  a few) versus a small number of people.
```

> * It has predictable and stable releases and has delivered on the release schedule for many years.

Reviewer 3: Is it better to call out specific year?
```suggestion
* It has predictable and stable releases and has delivered on the release schedule since 2015.
```

* It was a well defined security release process and manages security releases well.

> * As the defacto standard, it has the highest likely hood of being support for a given package on npm.

Reviewer 3: nit: typo
```suggestion
* As the defacto standard, it has the highest likelihood of being supported for a given package on npm.
```
